testdata/command-monitoring/pre-42-server-connection-id.json

```diff
@@ -98,4 +98,4 @@
       ]
     }
   ]
-}
```

Comment:
Was the new line removed accidentally or intentionally?

Reply:
I copied the contents of the [`monitoring`](https://github.com/mongodb/specifications/tree/master/source/command-logging-and-monitoring/tests/monitoring) directory in the specifications repo to our `testdata/command-monitoring` directory. Looks like their copy had no new line at the end and ours did. It shouldn't make a difference to our test runner.